plugins/minecraft/minecraft-users-ram_

```diff
@@ -21,9 +25,13 @@ then #is running
     MEMORYRSS=$(ps -p ${MC_PID} -o rss | cut -d'
 ' -f2)
     MEMGiB=$(echo "scale=2;${MEMORYRSS}/1024/1024" | bc -l)
+    CPUPERCENT=$(top -bp ${MC_PID} -n 1 | sed -n '$p' | tr -s ' ' | cut -d ' ' -f10)
+    CPU=$(echo "scale=2;${CPUPERCENT}/100" | bc -l)
```

Comment:
This line requires "bc" - this could be missing on some systems.
How about `awk '{printf $1/100}'` instead? (the same is valid for MEMGiB)
You should not resctrict the number of digits here - otherwise you loose precision for the averaging.